README.md

```diff
@@ -6,6 +6,14 @@
 
 This repository contains the Quarto source files of the practical exercises for the *Introduction to Social Attitudes Surveys* UKDS Data Skills Module.
 
+### The BSA
+
+A key aim of the BSA is to track the views and opinions of the public on national issues over time. Therefore, the BSA questionnaire is  designed to have several core questions that are repeated in most years. These cover different topics such as politics, welfare, poverty,  health, education, equalities, and employment.
+
+In addition, the interview questionnaire consists of various  background and demographic questions. The rest of the questionnaire  includes a series of non-core questions (modules) on a number of social, political, economic and moral topics, which are included in the survey  less frequently.
+
+The full questionnaire of the BSA survey used is available on the [documentation section](https://beta.ukdataservice.ac.uk/datacatalogue/series/series?id=200006#!/access-data) of the UK Data Service datasets for each survey year. [FAQs](https://beta.ukdataservice.ac.uk/datacatalogue/series/series?id=200006#!/faqs) are available.
```

Comment:
This is not necessary. Can you please remove?

Reply:
OK, I will remove